channels/offline-channel-js/Tests/Unit/src/dbprovider.tests.ts

```diff
@@ -65,44 +65,41 @@ export class OfflineDbProviderTests extends AITestClass {
             }
         });
 
-        this.testCaseAsync({
+        this.testCase({
             name: "IndexedDbProvider: init with auto clean set to true",
-            stepDelay: 100,
-            steps: [() => {
+            test: () => {
+                let endpoint = DEFAULT_BREEZE_ENDPOINT + DEFAULT_BREEZE_PATH;
                 let provider = new IndexedDbProvider();
                 let itemCtx = this.core.getProcessTelContext();
                 let storageConfig = createDynamicConfig({autoClean: true}).cfg;
                 let providerCxt = {
                     itemCtx:  itemCtx,
                     storageConfig: storageConfig,
-                    endpoint:DEFAULT_BREEZE_ENDPOINT + DEFAULT_BREEZE_PATH
-                }
-                this.ctx.isInit = provider.initialize(providerCxt);
-                this.ctx.provider = provider;
-
-                doAwait(provider.teardown(), () => {
-                    this.ctx.isclosed = true;
-                });
+                    endpoint: endpoint
+                };
 
-            }].concat(PollingAssert.createPollingAssert(() => {
-                let isInit = this.ctx.isInit;
-                if (isInit) {
-                    return true;
-                }
-                return false;
-            }, "Wait for init response" + new Date().toISOString(), 30, 1000) as any).concat(PollingAssert.createPollingAssert(() => {
-                let isclosed = this.ctx.isclosed;
-                if (isclosed) {
-                    return true;
-                }
-                return false;
-            }, "Wait for close response" + new Date().toISOString(), 30, 1000) as any)
+                return this._asyncQueue().add(() =>
+                    doAwait(provider.initialize(providerCxt), (val) => {
```

Comment:
Unless you create a local promise, when using `doAwait` it "should" return the result as it returns a promise which will make the caller (the queue) "wait" for it to finish before calling the next task, otherwise the next task will just run on the next execution cycle